js/blocks/BoxesBlocks.js

```diff
@@ -86,6 +86,32 @@ function setupBoxesBlocks() {
         }
     }
 
+    class DecrementOneBlock extends IncrementBlock {
+        constructor() {
+            super("decrementOne");
+            this.setPalette("boxes");
+            this.beginnerBlock(true);
+
+            this.setHelpString([
+                _("The Sub-1-from block subtracts one from the value stored in a box."),
+                "documentation",
+                ""
+            ]);
+
+            this.formBlock({
+                name: _("sub 1 from"),
```

Comment:
I think we should spell out subtract here.

Reply:
Done.